x-pack/plugins/cases/server/plugin.ts

```diff
@@ -57,17 +54,18 @@ export interface PluginsStart {
 
 export class CasePlugin {
   private readonly log: Logger;
-  private caseConfigureService?: CaseConfigureServiceSetup;
-  private caseService?: CaseServiceSetup;
-  private connectorMappingsService?: ConnectorMappingsServiceSetup;
-  private userActionService?: CaseUserActionServiceSetup;
+  private caseConfigureService?: CaseConfigureService;
+  private caseService?: CaseService;
+  private connectorMappingsService?: ConnectorMappingsService;
+  private userActionService?: CaseUserActionService;
   private alertsService?: AlertService;
+  private attachmentService?: AttachmentService;
   private clientFactory: CasesClientFactory;
   private securityPluginSetup?: SecurityPluginSetup;
   private config?: ConfigType;
 
   constructor(private readonly initializerContext: PluginInitializerContext) {
-    this.log = this.initializerContext.logger.get();
+    this.log = this.initializerContext.logger.get('plugins', 'cases');
```

Comment:
Does this change the order of these words in the log?

I think previously it was making the logs like this: `[case][plugins]`, does it now show up as: `[plugins][cases]`?

Reply:
I am not sure to be honest 😊